### PR TITLE
feat: add setup.py

### PR DESCRIPTION
**Pull request by mstefferson**
# Context

Add setup.py

# Changes

* Add setup.py
* Get it to work with docker

# Behaves Differently

* Able to pip install

# Untested

* Failing tests fixed in other PR

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM manifoldai/orbyter-ml-dev:3.2
+COPY requirements.txt /build/requirements.txt
+WORKDIR /build/
+RUN pip install -r requirements.txt
+WORKDIR /mnt/
+COPY setup.py tox.ini myproject.toml /mnt/
+COPY ../mlfaker /mnt/
```
Comment (jeffmm) [inline Dockerfile]: `COPY . /mnt/` ?

Comment (jeffmm) [reply Dockerfile]: Otherwise relying on owner to properly clone repo into the right directory

**Review by jeffmm — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (mstefferson) [reply Dockerfile]: good catch

**Pull request merged**